### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by varghesejose2020**
added my profile

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -520,5 +520,11 @@
     "githubUsername": "webdevrachit",
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
+  },
+  {
+    "name": "Varghese Jose",
+    "githubUsername": "varghesejose2020",
+    "imageUrl": "https://user-images.githubusercontent.com/71702982/106555821-526d2c00-6544-11eb-9da0-076566844959.JPG",
+    "issueNumber": 287
```
Comment (AllanRegush) [inline data.json:528]: What is the issue number? #287 is a Pull Request.

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1:

File: data.json (hunk 1 above)
Comment (varghesejose2020) [reply data.json:528]: What to fill in that portion

Comment (varghesejose2020) [reply data.json]: issue number updated

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank You

**Comment by varghesejose2020** [comment]
Thank you :+1:

**Pull request merged**